app/controllers/api/v1/base_controller.rb

```diff
@@ -38,6 +38,8 @@ def current_contact
       contact = Contact.find_by(salesforce_id: current_accounts_user['salesforce_contact_id'])
       if contact.blank?
         contact = SyncSalesforceContactsJob.perform_now(current_accounts_user['salesforce_contact_id'])
+        SyncSalesforceContactSchoolRelationsJob.perform_now
```

Comment:
would think `contact_id` should be passed in here - this could be a very long-running call otherwise

Reply:
Done. Updated BaseController to send a contact_id as well.